src/main/java/uk/gov/justice/digital/delius/controller/secure/StaffResource.java

```diff
@@ -182,4 +182,10 @@ public Caseload getCaseloadForStaff(
             .orElseThrow(() -> new NotFoundException(String.format("Staff member with code %s", staffCode))));
     }
 
+    @ApiOperation(value = "Return the list of heads of a specific probation delivery unit (aka borough)")
+    @GetMapping(path = "/staff/pduHeads/{pduCode}")
```

Comment:
picky one... should this have a capital H? I think normally we use dashes so pdu-heads

Reply:
I do agree on this, but I think they are all like this in community api. Otherwise yes defo.